LICENSE

```diff
@@ -1,213 +1,235 @@
+MIT License
+
```

Comment:
Add our own copyright here as well, we changed a lot of stuff, so it's partially our product as well(below in the GNU GPL)